tools/automatic_query_fixer/src/test/java/com/google/cloud/bigquery/utils/queryfixer/QueryTokenProcessorTest.java

```diff
@@ -0,0 +1,30 @@
+package com.google.cloud.bigquery.utils.queryfixer;
+
+import com.google.cloud.bigquery.utils.queryfixer.entity.IToken;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class QueryTokenProcessorTest {
+
+  private QueryTokenProcessor createService() {
+    return new QueryTokenProcessor(new BigQueryParserFactory());
+  }
+
+  @Test
+  public void convertSqlToTokens() {
+    QueryTokenProcessor tokenService = createService();
+    String sql = "Select col from `d1.t1`\n" +
```

Comment:
The auto formatting seems a bit weird. The line is too short...

Reply:
I break this on purpose. I want to simulate the BigQuery console, where a user can type a query in multiple lines.